examples/experimental/dagster-components/dagster_components/impls/dbt_project.py

```diff
@@ -0,0 +1,41 @@
+from dagster._core.definitions.definitions_class import Definitions
+from dagster._utils import pushd
+from dagster_dbt import DbtCliResource, DbtProject, dbt_assets
+from dagster_embedded_elt.sling.resources import AssetExecutionContext
+from pydantic import TypeAdapter
+from typing_extensions import Self
+
+from dagster_components import Component, ComponentLoadContext
+from dagster_components.core.component_decl_builder import ComponentDeclNode, YamlComponentDecl
+
+
+class DbtProjectComponent(Component):
+    params_schema = DbtCliResource
```

Comment:
Another case where I found it useful to slap the resource in as the params schema -- this will probably get nested underneath another key eventually, i.e.:

```yaml
component_type: ...

component_params:
  dbt_resource:
     project_dir: jaffle_shop
  other_stuff: ...
```

Reply:
Yes I think as convention it is critical to always have a top-level schema object that is extendible. The resource can be attached to that.